src/Components/IconEditorTab/IconEditorTab.scss

```diff
@@ -50,41 +59,38 @@
       line-height: 33px;
       padding: 2px 10px;
       border-radius: 4px;
-      font-size: 13px;
+      font-size: 14px;
       margin: 2px 0;
       .sr-only {
         @include srOnly();
       }
+      i,
       .bs-icon {
         margin-right: 10px;
-        color: var(--bs-primary, #5c9dcd);
+        color: #507ab5;
         font: normal normal normal 14px 'bootstrap-icons';
       }
       &:hover {
-        background-color: #fff;
+        background: #fafafa;
         color: #333;
         text-decoration: none;
+        box-shadow:
+          0 0 4px 1px rgba(0, 0, 0, 0.1),
+          0 0 0 2px #507ab5 inset;
         .bs-icon {
-          font-size: 23px;
-        }
-        .text-muted {
-          color: #bbe2d5;
+          font-size: 14px;
         }
       }
       &.active {
```

Comment:
Could you convert the `<a href="#"` tags in `AllIcons.tsx`, `IconSearch.tsx` and `SingleIcon.tsx` to `<button` tags? I assume, that would also need to adjust some stylings then.

Reply:
FIXED in #[3e3b10a]